### DISCUSSION TITLE
Resume

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#129] [General]
Hi I hope you're well, I was doing a 3d conversion with an extremely high depth resolution that gets it close to Blu Ray 3D quality . it was gonna take 7 days and then unfortunately my computer froze up , As a result it appears I had to reboot so I wanted to find out if there was any way that I could recover or resume from the file and if not I wanted to find out if u can consider adding a feature that automatically saves progress along the way similar to what owl3d does where in the Event you incur a power outage or you had to reboot your computer because it froze up. Adding a feature like this would allow you to resume from the last automatic save point without you having to lose the whole conversion and start the process all over. Thanks in advance As I believe this is a crucial and really important feature to have at least since these conversions take a long time and would really help mitigate risk of losing everything on the event a computer crash happens or power outage. I did see a resume in IW3 but it's grayed out and it's not clear on how to use it but Even so I'm not sure it does the auto save along the way I'm talking about like Topaz and  owl3d does which I still believe is a valid feature to add

**Comment by Ryudoadema** [comment]
My rendering PC is in my bedroom, so I turn it off every night. This would be super helpful so I don't have to use my potato in another room to use iw3. Thanks dev for your fast updates! I will get you some coffees soon :)

**Comment by nagadomi** [comment]
I am aware that this request has come in many times.
I will try to support it when there are no other high priority issues.

**Reply by kaelsonofkrypto** [reply]
thanks so much

**Comment by nagadomi** [comment]
Resume for Export/Export disparity is now supported.
When `Resume` is checked, Export process is resumed from a little before the last sequence existing in depth/rgb directory.
Note that there can be problems if the settings are different from previous runs.

**Reply by kaelsonofkrypto** [reply]
Thanks brother, appreciate you,
